src/screens/internal/AllStates/AllStates.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import { LoadingScreen } from './AllStates.style';
-import { useProjections } from 'common/utils/model';
+import { useProjectionsFromRegion } from 'common/utils/model';
 import { STATES } from 'common';
 import { Metric } from 'common/metric';
 import { MetricChart } from 'components/Charts';
+import regions from 'common/regions';
 
 function AllStates() {
-  return Object.keys(STATES).map(stateId => (
-    <State key={stateId} stateId={stateId} />
-  ));
+  return Object.keys(STATES).map(stateCode => {
+    return <State key={stateCode} stateCode={stateCode} />;
+  });
```

Comment:
It might be better to pass the `state` objects directly instead of passing state codes, getting the region and then getting the projection

```tsx
import regions  from 'common/regions';

function allStates() {
  return regions.states.map(state => (
    <State key={state.fipsCode} state={state} />
  );
}

// ...
function State({ state }) {
  const projections = useProjectionsFromRegion(state);
  // ...
}
```

Reply:
I'll give it a shot and see if it takes much effort!

Reply:
Question - are the sets of states the same between the two? I guess I can find that out pretty easily